source.go

```diff
@@ -230,6 +230,7 @@ type Source struct {
 	CodeVerification    *CodeVerificationFlow `json:"code_verification,omitempty"`
 	Created             int64                 `json:"created"`
 	Currency            Currency              `json:"currency"`
+	Customer            string                `json:"customer"`
```

Comment:
It is not technically expandable, and we're unlikely to change this so I choose a string property this time. If it became expandable, it would be a breaking change but I think it's fine.